examples/cisco/demo_interface.pp

```diff
@@ -104,6 +104,29 @@
       vrf                   => 'test1'
     }
 
+    # For private vlan
+    if platform_get() =~ /n(3|5|6|7|9)k/ {
+      cisco_vlan { '120':
+        ensure     => present,
+        private_vlan_type => 'primary',
+      }
+      cisco_vlan { '121':
+        ensure     => present,
+        private_vlan_type => 'isolated',
+      }
+      cisco_vlan { '120':
```

Comment:
This will generate the following error when you run the manifest.
```
Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: Cisco_vlan[120] is already declared in file /etc/puppetlabs/code/environments/production/modules/ciscopuppet/manifests/cisco/demo_interface.pp:109; cannot redeclare at /etc/puppetlabs/code/environments/production/modules/ciscopuppet/manifests/cisco/demo_interface.pp:117 at /etc/puppetlabs/code/environments/production/modules/ciscopuppet/manifests/cisco/demo_interface.pp:117:7 on node n3x-164.cisco.com
Warning: Not using cache on failed catalog
Error: Could not retrieve catalog; skipping run
```
      cisco_vlan { '121':
        ensure     => present,
        private_vlan_type => 'isolated',
      }
      cisco_vlan { '120':
        ensure     => present,
        private_vlan_type => 'primary',
        private_vlan_association => ['121'],
      }
```